docs/recipes/pantheon.md

```diff
@@ -45,7 +45,7 @@ git clone https://github.com/kalabox/lando.git
 cd lando/examples/pantheon
 
 # Clone drops-7 into a nested docroot
-git clone git@github.com:pantheon-systems/drops-7.git web
+git clone https://github.com/pantheon-systems/drops-7.git web
```

Comment:
Users don't need to add their own ssh key if Terminus does it as part of the process.